tests/pw/pages/productsPage.ts

```
@@ -1149,6 +1149,55 @@ export class ProductsPage extends AdminPage {
+                break;
+            case 'reviews':
+                if (otherOption.enableReview) {
+                    await this.toBeChecked(productsVendor.otherOptions.enableProductReviews);
+                } else {
+                    await this.notToBeChecked(productsVendor.otherOptions.enableProductReviews);
+                }
+                break;
+            default:
+                break;
+        }
+    }
```

Biome

[error] Duplicate class member name "addProductOtherOptions"



(lint/suspicious/noDuplicateClassMembers)